### DISCUSSION TITLE
Download a file in background even after closing the browser window

### DISCUSSION THREAD

**Discussion by NaveenK-Philips** [cefsharp/CefSharp#3714] [General]
Hi,

I have added Cefsharp browser in UserControl for my WPF application. Implemented IDownloadHandler for downloading a file.

Suppose I download a large file assume 4gb zip file, I select a download location on the dialog and download starts in background.

I will close the browser window immedietly assuming download will happen in background, but download will not happen in the background.
In current architecture as soon as browser window is closed, browser window objects gets disposed.

How do we download a file even after closing the browser window.
Is this achievable? Basically we do not want user to wait while downloading a large file by keeping browser window open.

**Comment by amaitland** [comment]
Already asked at https://magpcss.org/ceforum/viewtopic.php?f=18&t=18527

Please continue the conversation there.
